sleigh/sleigh-runtime/src/decoder.rs

```diff
@@ -221,6 +253,7 @@ impl Decoder {
                         // backtracking, then exit here and backtrack. If backtracking is disabled,
                         // then this must be an invalid instruction, but we continue with decoding
                         // to provide a partial decoding for debugging.
+                        tracing::error!("This is an invalid instruction!");
```

Comment:
This can be reached when backtracking is allowed, and will cause huge numbers of logs to be generated for specs that use backtracking heavily (e.g., ARM32 spec). So this log should be removed.